### PR TITLE
Created first attempt at Markdown version

### PR DESCRIPTION
**Pull request by jehb**
Addresses #3 - @berecka @semioticrobotic Please take a look at this for me.

### CODE REVIEW

File: Open_Decision_Framework-community-v1.0.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+  - Publish progress in an open place
+  - Provide regular updates to sponsors, customers, and stakeholders
+
+*Make it safe to voice concerns*
+
+  - Invite project team and collaborators to raise risks and concerns you've overlooked.
+  - Ask: What might prevent this project from succeeding? What concerns will your team have? What are we missing?
+  - Publish risk and limitations uncovered along the way
+
+* Conduct a premortem*
```
Comment (semioticrobotic) [inline Open_Decision_Framework-community-v1.0.md]: Spacing issue here?

File: Open_Decision_Framework-community-v1.0.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+===========================================================================
+
+  - I understand why the decision was made and how it aligns to Red Hat's strategy, goals, and mission.
+  - There was visibility to the business requirements, research, and evaluation criteria.
+  - The decision-making process was inclusive and transparent.
+  - Although I wasn't the decision maker, I was able to contribute to the process.
+  - I may not agree with the decision, but it's obvious that the decision makers understand Red Hat's values and culture.
+  - I might be disappointed, but I wasn't surprised.
+  - My voice was heard and valued.
+
```
Comment (semioticrobotic) [inline Open_Decision_Framework-community-v1.0.md]: Do we need a heading here to indicate that the phases are coming, or to "preview" the phases with a list before diving in?

File: Open_Decision_Framework-community-v1.0.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+  - [The Advice Process](http://danieltenner.com/2014/11/06/the-advice-process-definition-and-usage-tips/) (Daniel Tenner)
+
+APPENDIX 
+========
+
+History: Where the Open Decision Framework came from
+----------------------------------------------------
+
+  - Based on principles practiced by open source communities
+
```
Comment (semioticrobotic) [inline Open_Decision_Framework-community-v1.0.md]: Are these bullets nested? Spacing indicates they might be?

Comment (jehb) [inline Open_Decision_Framework-community-v1.0.md]: @semioticrobotic They were nested in the ODP, so I tried to preserve that formatting.

**Pull request merged**